language/move-analyzer/editors/code/tests/lsp-demo/sources/M6.move

```diff
@@ -0,0 +1,42 @@
+module Symbols::M6 {
```

Comment:
It seems like these test files are duplicates of those in `language/move-analyzer/tests/symbols/sources`. Could we keep just one copy?

I suggest we we adopt the following convention. If a move source file is used only for integration test, it would be in `language/move-analyzer/editors/code/tests/lsp-demo/sources`, but if it's used for both integration and unit tests, let's keep it in `language/move-analyzer/tests/symbols/sources`.

Reply:
I've been unable to reliably access files from `language/move-analyzer/tests/symbols/sources` via the test-runner in `editors/code`. 

Instead, I've stripped the irrelevant portions of code from the tests so they can be used purely as integration tests. Similar to what @yubing744 did for porting `M1.move` over as an integration test.

Reply:
It just rubs me the wrong way to have Move source files used for testing (and similar ones at that) in two different locations.

> I've been unable to reliably access files from `language/move-analyzer/tests/symbols/sources` via the test-runner in `editors/code`.

If the opposite is true (you can access `editors/code` tests from symbolicator tests, and if changing your tests to use the same files is not a huge problem, please use tests in `editors/code` for everything and I will move my tests there (will also try to re-do @yubing744's  integration test to use the original, possibly augmented, M1 module code).

Reply:
I can't access `editors/code` tests from symbolicator tests with the current setup because `cargo test` runs the tests at a different level compared to `editors/code`

This causes the reference to `MoveStdLib` to break (see: `Move.toml` in `editors/code`). If I update the path `MoveStdLib` path then the symbolicator tests start working but the `editors/code` tests break.

If you know how to fix that reference to the StdLib then I think what you're proposing will work!

Reply:
> If you know how to fix that reference to the StdLib in both cases then I think what you're proposing will work!

Technically we could pull the dependency remotely but perhaps it's indeed not worth it at the moment. Thank you for your patience!